authorize-service/src/authorize.rs

```diff
@@ -45,3 +46,28 @@ pub fn authorize<N: Network>(bytes: Bytes) -> Result<Value> {
         process.borrow().as_ref().unwrap().authorize(&bytes)
     })
 }
+
+pub fn authorize_signed<N: Network>(bytes: Bytes) -> Result<Value> {
```

Comment:
Would be good to make this use concrete types, `Request` and `Authorization<N>` instead of `Bytes` and `Value` and then do the serialization in the routes, if that's possible.

Reply:
My reasoning is because we probably will use our own service and just use this as a library